src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -47,8 +47,8 @@ export const breadcrumbs = () => (
             : (showLongItems = false)}
         <Heading level={1}>Breadcrumbs</Heading>
         <p>
-            Breadcrumbs background color matches the secondary section color of
-            the website section they're placed into. See more about brand colors{" "}
+            {"Breadcrumbs background color matches the secondary section color of " +
+                "the website section they're placed into. See more about brand colors "}
```

Comment:
This bothers me because I won't remember to write text like this just for prettier. But if it updates this on its own, I'm fine with it.

Reply:
It ... doesn't.  
Honestly, I kinda doing all that editing too, I'm happy to disable this and change these guys back.

Reply:
I will definitely have to google this each and every time if the editor doesn't do it automatically for us 😟

Reply:
The IDE might be able to point out the issue and also give a recommendation, but not sure if that's true for every case.

Reply:
The IDE does give a recommendation, but it's extra work that's frustrating I don't think it helps us much in the clean code department.

Reply:
It's muuuch too troublesome to go back and remove these, but I have disabled it in the ESLint config file and you can see that it works by the fact that this line no longer errors.